doc/make.py

```diff
@@ -48,6 +49,37 @@ def _generate_index(include_api, single_doc=None):
                          single_doc=single_doc))
 
 
+def _generate_exclude_pattern(include_api=True, single_doc=None):
```

Comment:
Not major, but the name "include_api" threw me off. Are we including it in output, or are we including it in the list of exclusions, thus excluding it? :)

Reply:
Yeah :) I agree it might be confusing, but it is the same keyword name as for `_generate_index`, and there it is more clear that `include_api` means to include it in the index.rst file (and thus here, to not put it in the excluded files)